webforj-foundation/src/main/java/com/webforj/optiondialog/OptionDialog.java

```diff
@@ -0,0 +1,127 @@
+package com.webforj.optiondialog;
+
+/**
+ * A utility class for creating and showing MessageBoxDialogs.
+ */
+public final class OptionDialog {
```

Comment:
I am not 100% happy with the class name. I kind of liked msgbox, as this name is known from quite a number of languages like VB, C#. I do get that the name is inspired by Swing, which also has value. Just sharing my thoughts here, I am under what we do with this.

Reply:
There may be a misunderstanding here. The `OptionDialog` class functions as a factory for blocking dialogs, which includes not only the BBj msgbox but also several other methods such as `prompt`, `filesave`, and `fileopen`.